src/js/popup.js

```diff
@@ -131,6 +133,11 @@ async function showRelayPanel(tipPanelToShow) {
   const { emailsBlockedVal } = await browser.storage.local.get("emailsBlockedVal");
 
 
+  //Subdomain Data
+  const { premiumSubdomainSetString } = await browser.storage.local.get("premiumSubdomainSet");
+  checkUserSubdomain(premiumSubdomainSetString);
```

Comment:
Oops, I may have given (MOAR!) bad advice here. I think the de-structuring assignment syntax wants the variable name to be the exact same name as the object property. 😢 

So I think it has to be:
```suggestion
  const {premiumSubdomainSet} = await browser.storage.local.get("premiumSubdomainSet");
  checkUserSubdomain(premiumSubdomainSet);
```

Reply:
Done! I added spaces around `{ premiumSubdomainSet }`, hope that's ok!